woocommerce/woocommerce-admin
# Homepage: Style SummaryNumbers

Fixes https://github.com/woocommerce/woocommerce-admin/issues/4084
Depends on https://github.com/woocommerce/woocommerce-admin/pull/4427

Styles the new SummaryNumbers shown on the Home screen as well as Overview and reports.

### Accessibility




### Screenshots

![Screen Shot 2020-05-26 at 3 20 49 PM](https://user-images.githubusercontent.com/1922453/82857330-c80c0b80-9f64-11ea-8edc-7cc6a31d3d20.png)

![Screen Shot 2020-05-26 at 3 23 58 PM](https://user-images.githubusercontent.com/1922453/82857405-0275a880-9f65-11ea-8af4-caca3e3786c7.png)

![Screen Shot 2020-05-26 at 3 24 19 PM](https://user-images.githubusercontent.com/1922453/82857408-043f6c00-9f65-11ea-8a8e-8ee9ef0b9c66.png)

![Screen Shot 2020-05-26 at 3 23 24 PM](https://user-images.githubusercontent.com/1922453/82857402-01447b80-9f65-11ea-9e38-ea7dbce28175.png)

### Detailed test instructions:

1. Visit Home screen, Reports, and Overview.
2. See the SummaryNumbers against various screen sizes and date ranges.
3. Ensure styles match designs in pbIJXs-4Z-p2

### Changelog Note:
